Assets/Scripts/BaseScript.cs

```diff
@@ -49,9 +51,22 @@ public void RegisterFailedLanding()
         mainCamera.transform.localPosition = new Vector3(0, 5.717994f, -990);
 
         // Reset rocket
-        IRocket newRocket = rocket.CreateRocket(transform, new Vector3(0.02f, -1.89f, -92f));
+        //IRocket newRocket = rocket.CreateRocket(transform, new Vector3(0.02f, -1.89f, -92f));
+
+
+        GameObject tempRocket = GameObject.Find("Rocket");
```

Comment:
This is awesome! I didn't think of swapping out the sprites like that, and I love that we can specify the sprite in the unity editor and have it pick from that!

Two quick thoughts though: 
- What do you think of having it pick a random number between 0 and rockets.Length? That way we can add sprites in the editor without making any code changes
- Would it be possible to move this logic in the Rocket script like you did with the asteroid logic? Unless you think there's a better place for it to live, I'm just not sure if having all this instantiation logic here is the best place for it

Reply:
I can try moving the randomization logic into the rocket script itself.